src/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+var _ = { extend: require('lodash.merge') };
+
+var rememberMe = module.exports = {
+  key: 'rememberMe'
+};
+
+rememberMe.set = function( params, errorCallback ) {
+  var defaults = {
+    url: location.href
```

Comment:
it is the default, but you can override it... I thought it was something we'd always want to save, what url you are on, but in my very first use of this I needed to save the url you are about to redirect to instead...